css-canvas-medical/index.html

```diff
@@ -0,0 +1,240 @@
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-half">
+        <div class="text-align">
+          <p>Temescal Creek Medicine, Oakland CA
+          </p>
+        </div>
+      </div>
+      <div class="column-half">
+        <div class="text-align">
+          <p>Caribou Medical, Towanda PA
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-half">
+        <div class="text-align">
+          <p>San Juan Healthcare, Friday Harbor WA
+          </p>
+        </div>
+      </div>
+      <div class="column-half">
+        <div class="text-align">
+          <p>Bay Tree Family Care, Red Bay AL
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+
+
+  <div class="container">
+    <div class="row">
+      <div class="column-half">
+        <h2 class="Ready-to-learn-font">Ready to learn more
+        </h2>
+        <p>Schedule a private meeting now
+        </p>
+      </div>
+      <div class="column-half">
+        <div class="text-align-right">
+          <button class="button-get-started">Get Started</button>
+        </div>
+      </div>
+    </div>
+  </div>
+
+
+  <div class="container" id="canvas-bottom">
+    <div class="row">
+      <div class="column-half">
+        <img class="logo-image" src="images/logo-white.png" alt="logo">
+        <p class="address">2307 Irving Street, Suite 228
+          <br>San Francisco CA 94122
+          <br>(800)370-1416
+          <br>info@canvasmedical.com
+        </p>
+      </div>
+      <div class="column-half">
+
+
+        <div class="row">
+          <div class="column-half">
+            <div>
+              <button class="button-bottom">COMPANY</button>
+            </div>
+          </div>
+          <div class="column-half">
+            <div>
+              <button class="button-bottom">PRODUCT</button>
+            </div>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-half">
+            <div>
+              <button class="button-bottom">About Us</button>
+            </div>
+          </div>
+          <div class="column-half">
+            <div>
+              <button class="button-bottom">Product</button>
+            </div>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-half">
+            <div>
+              <button class="button-bottom">Blog</button>
+            </div>
+          </div>
+          <div class="column-half">
+            <div>
+              <button class="button-bottom">Implementation</button>
+            </div>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-half">
+          </div>
+          <div class="column-half">
+            <div>
+              <button class="button-bottom">For Health Plan</button>
+            </div>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-half">
```

Comment:
make sure to remove empty elements like this, if you are using it for spacing, I would recommend going for the same result using flexbox